dev/benchmarks/test_apps/stocks/lib/i18n/stock_strings.dart

```diff
@@ -97,7 +97,7 @@ abstract class StockStrings {
   static const List<Locale> supportedLocales = <Locale>[
     Locale('en'),
     Locale('en', 'US'),
-    Locale('es'),
+    Locale('es')
```

Comment:
I believe the style guide asks to keep the trailing comma here, but double check me on that.